{
  "name": "eslint-plugin-no-unawaited-dot-catch-throw",
  "version": "0.0.3",
  "description": "ESLint plugin to enforce that unawaited .catch( promises are caught and re-thrown",
  "main": "eslint-plugin-no-unawaited-dot-catch-throw.js",
  "scripts": {
    "test": "node enforce-nounawaited-dot-catch-throw.test.js"
  },
  "peerDependencies": {
    "eslint": ">=7.32.0"
  },
  "keywords": ["eslint", "eslintplugin", "eslint-plugin", "catch", "unawaited", "throw"],
  "author": "Reece Daniels <rubengmurray@gmail.com>",
  "license": "ISC",
  "devDependencies": {
    "eslint": "^7.32.0"
  },
  "homepage": "https://github.com/rubengmurray/no-unawaited-dot-catch-throw#readme",
  "repository": {
    "type": "git",
    "url": "https://github.com/rubengmurray/no-unawaited-dot-catch-throw.git"
  },
  "engines": {
    "node": ">=18.13.0"
  }
}
